PDF2Text/script.py

```diff
@@ -0,0 +1,13 @@
+import PyPDF2
+
+pdf = input(r"Enter the path of PDF file: ")
+n = int(input("Enter number of pages: "))
+st=""
+page = PyPDF2.PdfFileReader(pdf) 
+for i in range(n):
+    
+    st += page.getPage(i).extractText()
```

Comment:
You need to store each page individually. That means, if I have a pdf of 100 pages, then 100 text files must be created, one file for one page of PDF.

Reply:
@kaustubhgupta  can you suggest me the code for separately storing the pdf

Reply:
as you are getting the page numbers from the user, you can write a for a loop until that number, then in that for loop, use the with-open pair of file handling to store that text as txt files and name them as 1,2,3 and so on. Naming will be easier as you can use current iteration number as file name (using f-strings)